### PR TITLE
Feature/array validator

### PR DESCRIPTION
**Pull request by brauliodiez**

### CODE REVIEW

File: form-libraries/react-final-form.mdx [hunk 1]
```diff
@@ -208,3 +208,36 @@ We can define the validation for the _product.name_ nested property by using the
...
 - [SourceCode](https://github.com/Lemoncode/fonk/tree/master/examples/react-final-form/ts/nested-field)
+
+## Array field validations
+
+Sometimes you need to apply validations to a collection of items (e.g. in a shopping cart list, check that each row quantity field is informed and a valid numeric value),
+you can do that using the built in _ArrayValidator_
+
+### Example A - Shopping cart
+
+![array-shopping-cart-validation](../gifs/formik/array-shopping-cart-validation.gif)
```
Comment (Nasdan) [inline form-libraries/react-final-form.mdx:219]: Use  `../gifs/react-final-form/...`

File: form-libraries/react-final-form.mdx [hunk 2]
```diff
@@ -208,3 +208,36 @@ We can define the validation for the _product.name_ nested property by using the
...
+- [SourceCode](https://github.com/Lemoncode/fonk/tree/master/examples/react-final-form/js/shopping-cart-array-validator)
+
+**Typescript**
+
+- [CodeSandbox](https://codesandbox.io/s/github/lemoncode/fonk/tree/master/examples/react-final-form/ts/shopping-cart-array-validator)
+- [SourceCode](https://github.com/Lemoncode/fonk/tree/master/examples/react-final-form/ts/shopping-cart-array-validator)
+
+### Example B - Adding multiple users (EMail verification)
+
+![array-create-multiple-users](../gifs/formik/array-create-multiple-users.gif)
```
Comment (Nasdan) [inline form-libraries/react-final-form.mdx:233]: Use  `../gifs/react-final-form/...`

File: validators/builtin-validators.mdx [hunk 3]
```diff
@@ -287,6 +288,55 @@ Example:
 
 You can find this example in [Javascript](https://github.com/Lemoncode/fonk/tree/master/examples/docs/api/js/email) and [Typescript](https://github.com/Lemoncode/fonk/tree/master/examples/docs/api/ts/email).
 
+## Array
+
+This validator allows you to add support to validation to array form fields, it work as an inception,
```
Comment (Nasdan) [inline validators/builtin-validators.mdx:293]: Add s in -> it **works** as an inception,

File: validators/builtin-validators.mdx [hunk 4]
```diff
@@ -287,6 +288,55 @@ Example:
...
+you get the fonk engine embedded into that array, _e.g. you got a shopping cart list and you want
+to validate that each Quantity field on each row is informed and is a valid number_.
+
+```typescript
+export interface ArrayArgs {
+  field: FieldValidationSchema;
+}
+
+interface ArrayValidationResult {
```
Comment (Nasdan) [inline validators/builtin-validators.mdx:302]: Remove from 302 to 312 lines, it's an internal interface

File: form-libraries/react-final-form.mdx (hunk 1 above)
Comment (brauliodiez) [reply form-libraries/react-final-form.mdx:219]: done

File: form-libraries/react-final-form.mdx (hunk 2 above)
Comment (brauliodiez) [reply form-libraries/react-final-form.mdx:233]: done

File: validators/builtin-validators.mdx (hunk 3 above)
Comment (brauliodiez) [reply validators/builtin-validators.mdx:293]: done

File: validators/builtin-validators.mdx (hunk 4 above)
Comment (brauliodiez) [reply validators/builtin-validators.mdx:302]: done

**Pull request merged**